### ISSUE TITLE
Accessibility: ListViewGroup Subtitle Property

### ISSUE THREAD

**Issue by lonitra** [dotnet/winforms#3261]
**Problem description:**

The ListViewGroup has a subtitle property that is not being picked up by both inspect and narrator.

**Expected behavior:**

The subtitle property should be read by narrator and it should also be displayed somewhere in inspect

**Minimal repro:**

1. Create a winforms core app
2. Add a ListView and ListViewGroup with a subtitle
4. Run app
3. Start inspect/narrator and click on the ListViewGroup with the subtitle in the app

**Note:**
This issue stems from PR #3234

**Comment by MandiMan** [comment]
Verified on .NET 7.0 test pass build: .NET 7.0.100-rc.1.22425.12, issue was fixed as below:
![listview](https://user-images.githubusercontent.com/108860782/187168000-9911116b-0367-4c10-bd1c-d2c74931fa55.png)

**Comment by MelonWang1** [comment]
Verified on .NET 8.0 latest build from main branch: NET 8.0.100-alpha.1.22451.15, issue was fixed, test result is same as above screenshot.
